static/js/ui_report.js

```diff
@@ -44,7 +44,12 @@ exports.error = function (response, xhr, status_box, type) {
     if (xhr && xhr.status.toString().charAt(0) === "4") {
         // Only display the error response for 4XX, where we've crafted
         // a nice response.
-        response += ": " + escape(JSON.parse(xhr.responseText).msg);
+        var response_text = escape(JSON.parse(xhr.responseText).msg);
```

Comment:
I'm not positive we need this, but if we do, it should probably be a separate commit.

Reply:
The reason I have add this is to prevent the notification message which include more than one `:`. 
The cases can be:
* In most of error notification we have passed, `Failed` or `Error` or `Error updating setting` text as extra message in notification
* So our final notification message will be 
`extra_message`: `xhr.responseText`

Now, suppose I have passed `Error` as extra message and `xhr.responseText` is `Invalid domain: Domain must have at least one dot(.)`
In this case final message will be like
`Error: Invalid domain: Domain must have at least one dot(.)`
So that's why I add condition to check whether `xhr.responseText` include `:` or not.